spec/features/user_signs_out_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-feature 'user signs out', %Q{
+feature 'user signs out', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.